addons/medical_gui/stringtable.xml

```diff
@@ -44,27 +47,35 @@
         </Key>
         <Key ID="STR_ACE_Medical_GUI_EnableSelfActions_DisplayName">
             <English>Enable Medical Self Actions</English>
+            <German>Medizinische Selbst-Interaktionen anzeigen</German>
         </Key>
         <Key ID="STR_ACE_Medical_GUI_EnableSelfActions_Description">
             <English>Enables medical actions for the Self Interaction Menu.</English>
+            <German>Medizinische Interaktionen bei Selbst-Interaktionen anzeigen</German>
         </Key>
         <Key ID="STR_ACE_Medical_GUI_EnableMedicalMenu_DisplayName">
             <English>Enable Medical Menu</English>
+            <German>Aktiviere das Sanitätsmenü</German>
         </Key>
         <Key ID="STR_ACE_Medical_GUI_EnableMedicalMenu_Description">
             <English>Enables the use of the Medical Menu through the keybind or interaction menu.</English>
+            <German>Aktiviere die Nutzung des Sanitätermenüs durch eine Tastenkombination oder des Interaktionsmenüs</German>
```

Comment:
in L58 it's "Sanitätsmenü" and here it's "Sanitätermenü"

Reply:
I prefer "Sanitätsmenü".
Because otherwise Sanitäterinnen might complain.

Reply:
Yep you are right